### DISCUSSION TITLE
How to extract I3D features on different span and stride

### DISCUSSION THREAD

**Discussion by rabeya-akter** [v-iashin/video_features#102] [Q&A]
TSPNet/
├── i3d-features/
│   ├── span=8_stride=2
│   ├── span=12_stride=2
│   └── span=16_stride=2
├── data-bin/
│   └── phoenix2014T/
│       └── sp25000/
│   
├── README.md
├── run-scripts/
└── test-scripts/

I want to extract i3d features on 3 different span (span = 8 _ stride= 2, span = 12_ stride = 2, span = 16 _ stride =2), how can I do this using your i3d feature extraction code. Can you please guide me?

**Comment by v-iashin** [comment]
https://iashin.ai/video_features/models/i3d/

I am not dure what you are referring to. Perhaps, span=stack_size; stride=step_size
